README.md

```diff
@@ -14,14 +14,31 @@ To use this package, add the following line to your `.emacs` file:
     (require 'auctex-latexmk)
     (auctex-latexmk-setup)
 ```
-And add the following line to your `.latexmkrc` file:
-```perl
-    # .latexmkrc starts
-    $pdf_mode = 1;
-    # .latexmkrc ends
-```
 After that, by using `M-x TeX-command-master` (or C-c C-c), you can use
 LatexMk command to compile TeX source.
+    
+LatexMk will inherit many AUCTeX settings, including:
+* Run with `-interaction-nonestopmode` if `TeX-interactive-mode` minor mode is
+  active
+* Run with `-synctex` if `TeX-source-correlate-mode` is active
+  
+If you would like LatexMk to aso pass the `-pdf` flag when `TeX-PDF-mode` is
+active add
+```elisp
+    (setq auctex-latexmk-inherit-TeX-PDF-mode t)
```

Comment:
I'm not at all sure using `custom-set-variables` is a good idea. Personally I only use the customize interface to change things in `custom-set-variables`. I don't think suggesting that people edit it by hand is a good idea. I also don't think there is any downside to using `setq` in this case. 

It might be a good idea to mention in the README that there is a `auctex-latexmk` customize group and that you can set variables using `M-x customize-group auctex-latexmk RET`. I'll be happy to add something along those lines if you think its a good idea.